peerdid/peer_did.py

```diff
@@ -51,7 +86,14 @@ def resolve_peer_did(peer_did: PEER_DID, version_id=None) -> JSON:
     version_id is not used for now, as we support only static layer where did doc never changes
     :return: resolved did_doc as a JSON string
     """
-    pass
+    if is_peer_did(peer_did):
```

Comment:
Please avoid such long indentation. 
It would be much cleaner to write
```
if not is_peer_did(peer_did):
    raise ValueError('Wrong Peer DID')
if peer_did[9] == '0':
   ....
```